### PR TITLE
merges graphql service under federated service

### PR DESCRIPTION
**Pull request by kotharironak**
The changes include,
- adds hypertrace-graphql as sub-module
- embed graphql service into federated service
- graphql service runs its own port as its starts its own jetty server.

### CODE REVIEW

**Comment by kotharironak** [comment]
Looking at the build problem, locally it had worked are recursive pull.

File: hypertrace-federated-service/src/main/java/org/hypertrace/federated/service/FederatedService.java [hunk 1]
```diff
@@ -112,22 +119,28 @@ private Config getServiceConfig(String serviceName) {
...
         throw new RuntimeException(e);
       }
-      server.awaitTermination();
-    } catch (InterruptedException e) {
-      Thread.currentThread().interrupt();
-      throw new RuntimeException(e);
-    }
+    });
+    thread.start();
+    graphQlService.start();
```
Comment (adriancole) [inline hypertrace-federated-service/src/main/java/org/hypertrace/federated/service/FederatedService.java:137]: note: eventually if we use something like armeria we won't need to do things like this as they can live on the same port.

**Review by adriancole — APPROVED** [review]
worked locally. thanks!

**Review by JBAhire — APPROVED** [review]
tested and working!

**Comment by adriancole** [comment]
Merged as this is only used in our docker file, and we already had 3 different people test it, so no need to nag others. Thanks tons @kotharironak!

**Pull request merged**